Sources/XMTP/ConversationV2.swift

```diff
@@ -188,8 +193,8 @@ public struct ConversationV2 {
 		}
 	}
 
-	public var createdAt: Date {
-		Date(timeIntervalSince1970: Double(header.createdNs / 1_000_000) / 1000)
+	public var conversationCreatedAt: Date {
+		Date(timeIntervalSince1970: Double((createdAt ?? 0) / 1_000_000) / 1000)
```

Comment:
I think if you don't have the parent createdAt then you should use the header
```suggestion
		Date(timeIntervalSince1970: Double((createdAt ?? header.createdNs) / 1_000_000) / 1000)
```